generator/src/main/java/com/graphicsfuzz/generator/tool/Generate.java

```diff
@@ -202,7 +213,42 @@ public static StringBuilder generateVariant(ShaderJob shaderJob,
     }
 
     if (args.getGenerateUniformBindings()) {
-      shaderJob.makeUniformBindings();
+      boolean supportedInAllShaders = true;
```

Comment:
Change name to `pushConstantsSupportedInAllShaders` to be more descriptive.

Reply:
done